docs/source/model_design/risk_factors.rst

```diff
@@ -48,8 +48,111 @@ Definitions
+
+In GBD, relatives risks are usually ratio of incidence rates of causes in those exposed vs unexposed to the risk factor. However, there are exceptions as in the low birth rate short gestation (LBWSG) risk factor where the relative risks are ratios of all-cause mortality rates. It is best practice to always check with the risk appendix or the GBD modeller what the relative risks refer to each risk-outcome pair. 
+
+For example, a study was conducted to measure the effect of vitamin A food fortification on incidence of measles in children under 5. GBD defines risk factors to be malignant. Hence, the exposed group (exposed to poor nutrition) are those who are not covered by food fortification while those unexposed are covered by food fortification. The table below shows the results: 
+
+.. image:: rate_2x2table.svg
+
+rate1 is the rate disease in the exposed group (no fortified foods)
+rate0 is the rate of disease in the unexposed group (with fortified foods)
+The rate ratio (RR) is thus rate1/rate0, = 100/40 = 2.5. 
+
+This is interpreted as: 'children who do not eat foos fortified by vitamin A food are 2.5 times more likely to get measles than children who eat vitamin A enriched foods'.
+
+Alternatively, the risk is computed as follows:
+
+.. image:: risk_2x2table.svg
+
+risk1 is the risk of having measles in the exposed: a/(a+c) 
+risk0 is the risk of having measles in the unexposed: b/(b+d)
+The risk ratio is (RR) is thus risk1/risk0 = [a/(a+c)]/[b/(b+d)]
+
+This is interpreted as: 'there are X times more cases of measles among children who do not eat vitamin A fortified foods than those who eat vitamin A fortified foods'
+
+If we want to compute the odds ratio:
+The odds of disease in the exposed is a/c = risk1(1-risk1)
+The odds of disease in the unexposed is b/d = risk0(1-risk0)
+The odds ratio is: ad/bc= risk1(1-risk1)/risk0(1-risk0)
+
+If the disease is rare and not recurrent, then the risk ratio, the rate ratio and the odds ratio are numerically similar. Odds ratios are often derived from case-control studies in which people with and without the outcome of interest are compared for their exposure. Depending on how the controls were sampled the odds ratio in a case control study can be equivalent to the risk of rate ratios that would have been obtained if the whole population had been studied. 
```

Comment:
I really like @alibow's description, during epi group work last week, of why one might want to use odds

you were describing a scenario when you're looking at a rare condition/disease, for which you perhaps don't want to have to collect a sample size of 2 million people, so you go and find some number of with-condition people, and look at how many have the exposure, as compared to a random sample of without-condition people, and the proportion of that group with the exposure.

(pls correct if wrong)

that _really_ clarified odds for me, and might fit nicely here!

Reply:
done!